libraries/debug-certificate-manager/src/CertificateManager.ts

```diff
@@ -120,6 +121,14 @@ export class CertificateManager {
 
     const { certificateData: existingCert, keyData: existingKey } = this._certificateStore;
 
+    if (process.env.DISABLE_DEV_CERT_GENERATION === '1') {
```

Comment:
```suggestion
    if (process.env.RUSHSTACK_DISABLE_DEV_CERT_GENERATION === '1') {
```

And put the env var name in a constant?